Koryntia/KoryntiaWebApp
# add mongodb + loan CRUD

To test: 
- Make sure you have [MongoDB Compass](https://www.mongodb.com/products/tools/compass) up and running.
- Copy the content of`.env.example` and save it in `.env.local`
- Spin up the dev server,

	```
	pnpm i && pnpm dev
	```
- visit: http://localhost:3000/loans

Note:
- The `loans` route is only for testing purposes. Might/will be deleted later.
